pystarport/cluster.py

```diff
@@ -1057,8 +1057,9 @@ def supervisord_ini_group(chain_ids):
     }
     cfg["program:relayer-demo"] = dict(
         COMMON_PROG_OPTIONS,
-        command=("hermes --config %(here)s/relayer.toml start"),
-        stdout_logfile="%(here)s/relayer-demo.log",
+        directory="%(here)s",
+        command="hermes --config relayer.toml start",
+        stdout_logfile="relayer-demo.log",
```

Comment:
miss directory, but why need change replayed one?

Reply:
I haven't tested, we can revert this

Reply:
The changes are being tested here: https://github.com/crypto-org-chain/cronos/pull/722
If the integration tests success, I guess this PR is safe to merge.